spec/spec_helper_acceptance_local.rb

```diff
@@ -37,6 +37,8 @@ def print_parsing_errors
     LitmusHelper.instance.run_shell('setenforce 0', expect_failures: true) if %r{redhat|oracle}.match?(os[:family])
```

Comment:
This looks odd because It thought Oracle would be in the Red Hat OS family.

Reply:
Thanks @ekohl , I will make those changes too with this PR. Changed to 'name' from 'family' since this is only required for Redhat-7. Tested locally and its working fine.
`
  40) apache::vhost define proxy_pass_match for alternative vhost Service "httpd"
     # FM-8483
     # ./spec/acceptance/vhost_spec.rb:623

  41) apache::vhost define proxy_pass_match for alternative vhost Service "httpd"
     # FM-8483
     # ./spec/acceptance/vhost_spec.rb:624


Finished in 29 minutes 48 seconds (files took 10.23 seconds to load)
417 examples, 0 failures, 41 pending
`